src/pytest_workflow/plugin.py

```diff
@@ -427,10 +424,8 @@ def runtest(self):
         # workflow.exit_code waits for workflow to finish.
         assert self.workflow.exit_code == self.desired_exit_code
 
-    def repr_failure(self, excinfo):
-        message = ("'{workflow_name}' exited with exit code " +
-                   "'{exit_code}' instead of '{desired_exit_code}'."
-                   ).format(workflow_name=self.workflow.name,
-                            exit_code=self.workflow.exit_code,
-                            desired_exit_code=self.desired_exit_code)
+    def repr_failure(self, excinfo, style=None):
```

Comment:
This style thing is from pytest.Item? Since it isn't getting used.

Reply:
Yes this needs to match, otherwise the function signature is incorrect. `excinfo` also is not used, but it needs to be there.